src/main/java/util/Scene.java

```diff
@@ -24,7 +30,7 @@
      * Entry point to start the application
      */
     public static void main(String[] args) {
-        Engine.init(1600, 900, "Hello World!");
+        Engine.init(1600, 900, "Hello World!", 1);
```

Comment:
This is now not necessary because the init method defaults to 1 if no value is provided.